benchmarks/python/test_huggingface_attn_bwd.py

```diff
@@ -55,9 +55,15 @@ def huggingface_attn_bwd_fusion(
     fd.add_output(T35)
 
 
+def huggingface_attn_bwd_iobytes(size: tuple, dtype: torch.dtype):
```

Comment:
tiny nit: when annotating collections, specify the element types. For tuple, annotate all of them like `size: tuple[int, int, int, int]`.

Reply:
As @liqiangxl already mentioned, if this just implements the default behavior i.e. `sum(i.nbytes for i in inputs) + sum(o.nbytes for o in outputs)` then we should remove this helper.

Reply:
This is for uniformity of IObytes between nvFuser and torch when the inputs/outputs differ between the two. See the discussion at: https://github.com/NVIDIA/Fuser/pull/1725. I'll add a reference to this PR in a comment.

Reply:
> tiny nit: when annotating collections, specify the element types. For tuple, annotate all of them like `size: tuple[int, int, int, int]`.

I think for size, the element types are understood. So I prefer this to reduce verbosity. Not a strong preference, though. I may have a bias being more familiar with the benchmarks.

Reply:
Does the IOBytes issue affect this computation though? It seems like the helper method is just replicating default behavior. If that's the case, then maybe putting argument `iobytes=None,` would be the clearest. As you wish on the tuple annotation.

Reply:
The nvFuser input/outputs are grad_out, attention_mask, dropout_mask and grad_input.
The eager input/outputs are grad_out and output. So there will be a difference in the computed IObytes.

Reply:
Oh ok, so it does affect this case due to recomputation. Could you place a comment in the code describing that? Thanks.